terraform.tf

```diff
@@ -1,11 +1,13 @@
 terraform {
-  backend "s3" {}
+  backend "s3" {
+    bucket = "project-terraform-state-dev-allen"
```

Comment:
this file shouldn´t be updated. By default you have to use `terraform_exec.sh` file

Reply:
updated